src/libraries/System.Text.Json/src/System/Text/Json/Nodes/JsonObject.cs

```diff
@@ -248,9 +248,8 @@ internal void SetItem(string propertyName, JsonNode? value)
 
             OrderedDictionary<string, JsonNode?> dict = Dictionary;
 
-            if (!dict.TryAdd(propertyName, value))
+            if (!dict.TryAdd(propertyName, value, out int index))
```

Comment:
Nice. Would it be possible to scan the `JsonObject` for other uses of `TryAdd` and `TryGetValue` that might benefit from using the new overloads?

Reply:
There were no other instances of this pattern (not many uses of `TryAdd` and `TryGetValue` in general).